jsonschema-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/jsonschema/typeconverters/TypeConverterFactory.java

```diff
@@ -50,7 +50,12 @@ public TypeConverter get(@NonNull org.everit.json.schema.Schema jsonSchema,
         if (jsonSchema instanceof BooleanSchema) {
             typeConverter = get(Schema.Type.BOOLEAN);
         } else if (jsonSchema instanceof NumberSchema) {
-            typeConverter = get(Schema.Type.valueOf(connectType.toUpperCase()));
+            // If no connect type passed then assume that connect schema is for INT64 type data
+            if (connectType == null) {
+                typeConverter = get(Schema.Type.valueOf("INT64"));
```

Comment:
Why is it so?

Reply:
If no connect type passed then assume that connect schema is for INT64 type data. This is for cases where schema is not enriched schema created by GSR converters.

Reply:
What's the reasoning behind INT64?